### PR TITLE
HEG sytle update

### PR DESCRIPTION
**Pull request by grolimur**
updates based on the latest version of the ISO 690 standard, with improvements

### CODE REVIEW

File: iso690-author-date-en.csl [hunk 1]
```diff
@@ -118,7 +118,7 @@
       <if type="book thesis map motion_picture song manuscript" match="any">
         <choose>
           <if variable="author editor translator" match="any">
-            <text variable="title" font-style="italic"/>
+            <text variable="title" font-style="italic" suffix=". "/>
```
Comment (adam3smith) [inline iso690-author-date-en.csl:121]: Are you making the changes to all the other ISO-690 styles on purpose? (the PR title only mentions HEG). And if so, what's the idea of adding all the periods in the title suffix here? The title macro has period in the suffix every time it's called in the bibliography, so this shouldn't be necessary (and makes the style harder to maintain)

Comment (grolimur) [reply iso690-author-date-en.csl]: Thanks @adam3smith. You are right. I removed these periods in the title macro.

**Comment by adam3smith** [comment]
did you close this on purpose?

**Comment by grolimur** [comment]
No, sorry. I closed it accidentslly when I answered your question.

**Comment by grolimur** [comment]
@adam3smith Should I reopen it in order for this pull request be to taken in account in the next Zotero update?

**Pull request closed without merging**